app/interactors/cangaroo/perform_jobs.rb

```diff
@@ -15,7 +16,18 @@ def data
     end
 
     def enqueue_jobs(type, payload)
-      initialize_jobs(type, payload).select(&:perform?).each(&:enqueue)
+      initialize_jobs(type, payload).each do |job|
+        if job.perform?
+          # TODO log simplified info about payload info is being queued
+          job.enqueue
+        else
+          # TODO persist translation for audit trail?
```

Comment:
@iloveitaly Can you explain how do you plan to use translation on the logging class?